docs/core/additional-tools/xmlserializergenerator-instructions.md

```diff
@@ -0,0 +1,45 @@
+ # Using Microsoft Xml Serializer Generator on .NET Core
+
+Like the Xml Serializer Generator (sgen.exe) on desktop, Microsoft.XmlSerializer.Generator NuGet package is the solution for .NET Core and .NET Standard Libraries. It creates an Xml serialization assembly for types contained in an assembly to improve the startup performance of Xml serialization when serializing or de-serializing objects of those types using XmlSerializer. 
+
+You can start using the tool today following the instructions below. 
+
+## Prerequisites
+
+The following is required for Microsoft.XmlSerializer.Generator to work. You can use command `dotnet --info` to check which versions of .NET Core SDK and runtime you may already have installed.
+
+* [.NET Core SDK 2.0.2 or later](https://www.microsoft.com/net/download/windows)
+* [.NET Core runtime 2.0.3 or later](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.0.3.md)
```

Comment:
doesn't the SDK already come with the right runtime versions? Do we need to list the runtime separately?

Reply:
No, the SDK and the runtime are two different things.

Reply:
I know they are different things. But what I'm asking is, when you install the right SDK, doesn't that come bundled with the right runtime?

Reply:
SDK 2.0.2 would install runtime 2.0.2. The tool requires runtime 2.0.3.

SDK 2.0.3 is not released yet.

Reply:
Ok, then it makes sense to list them separately. Thanks @shmao

Reply:
@shmao -- Current .NET SDK is 2.1.3
It contains:
    • .NET Core SDK 2.1.3
    **.NET Core Runtime 2.0.4**
    • Runtime Package Store
Will that be ok?

Reply:
Current .NET Core SDK version is 2.1.3, which contains .NET Core Runtime 2.0.4.  I've modified to say .NET Core SDK 2.1.3 or later. Ok?